Assets/Scripts/Customization/HairCustomization.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[ExecuteInEditMode]
+public class HairCustomization : MonoBehaviour 
+{
+    public static HairCustomization instance;
+
+    public bool IsCreatingCharacter = false;

```

Comment:
false is default value for boolean, Don't define default compiler will do this for you.